pkg/koordlet/resmanager/metrics_query.go

```diff
@@ -109,16 +108,16 @@ func (r *resmanager) collectContainerThrottledMetricLast(containerID *string) me
 			QueryResult: metriccache.QueryResult{Error: fmt.Errorf("container is nil")},
 		}
 	}
+
 	queryParam := generateQueryParamsLast(r.collectResUsedIntervalSeconds * 5)
 	queryResult := r.metricCache.GetContainerThrottledMetric(containerID, queryParam)
 	if queryResult.Error != nil {
 		klog.V(5).Infof("get container %v throttled metric failed, error %v", *containerID, queryResult.Error)
-		return queryResult
```

Comment:
we may keep this return so that only one log in L115 will be printed when an error occurs. same below

Reply:
done